azure-devops/azext_devops/dev/repos/pull_request.py

```diff
@@ -306,43 +308,18 @@ def update_pull_request(id, title=None, description=None, auto_complete=None,  #
             pr.auto_complete_set_by = IdentityRef(id=EMPTY_UUID)
     if draft is not None:
         pr.is_draft = draft
+    if status is not None:
+        pr.status = status
+        if status == 'completed':
+            pr.last_merge_source_commit = existing_pr.last_merge_source_commit
+            pr.completion_options = existing_pr.completion_options
```

Comment:
This should be pr.completion options now. If I update the completion options and status in the same update command you will end up overwriting the completion options after setting them in lines 289-303.

Reply:
nice catch

Reply:
that is being set selectively so you will need to handle appropriately. 

When the update is to set the status as completed. PR code on server needs the completion options to be sent with the request.